cloudsplaining/output/src/components/finding/PolicyDocumentDetails.vue

```diff
@@ -4,15 +4,18 @@
         <div class="card-header">
             <a class="card-link" data-toggle="collapse"
                v-bind:data-parent="'#' + inlineOrManaged.toLowerCase() + '-policy' + '-' + policyId + '-' + 'card-details'"
-               v-bind:href="'#' + inlineOrManaged.toLowerCase() + '-policy' + '-' + policyId + '-' +'policydocument'"
-            >Policy Document</a>
+               v-bind:href="'#' + inlineOrManaged.toLowerCase() + '-policy' + '-' + policyId + '-' +'policydocument'">
+               Policy Document
+            </a>
         </div>
-        <div class="panel-collapse collapse"
-             v-bind:id="inlineOrManaged.toLowerCase() + '-policy' + '-' + policyId + '-' +'policydocument'">
+        <div 
+            class="panel-collapse collapse"
+            ref="PolicyDocumentDetailsDiv"
+            v-bind:id="inlineOrManaged.toLowerCase() + '-policy' + '-' + policyId + '-' +'policydocument'">
             <div class="card-body">
-    <pre><code>
-    {{ JSON.parse(JSON.stringify(policyDocument(policyId), undefined, '\t')) }}
-    </code></pre>
+                <pre><code>
+                    {{ JSON.parse(JSON.stringify(policyDocument(policyId), undefined, '\t')) }}
```

Comment:
Please remove this whitespace before the `                    {{ JSON.parse(JSON.stringify(policyDocument(policyId), undefined, '\t')) }}`. 

That make it look like this: 
![image](https://user-images.githubusercontent.com/3422255/97234728-cff00600-17b7-11eb-89bb-91e8fddbfcad.png)

Once you remove that, regenerate the JavaScript with `npm build`, and regenerate the report, I can merge.

Reply:
Removed the whitespace